README.md

```diff
@@ -26,7 +26,16 @@ editor.withoutNormalizing(() => {
 editor.moveToRangeOfDocument()
 ```
 
-### Analyzing 
+#### Save and Restore Selection
+```
+// Save selection before focusing on something else
+const selection = editor.value.selection
+
+// Restore editor selection later on
+editor.change(change => change.select(selection))
```

Comment:
Should this be `editor.select(selection)` in the latest version?

Reply:
You're right, give me a sec.

Reply:
You're right, give me a sec.

Reply:
Time to update my own codebase too. :+1: